protos/sources/blockexplorer/api/v1/blockexplorer.proto

```diff
@@ -65,8 +65,18 @@ message ListTransactionsRequest {
   optional string before = 2;
   // Optional cursor to paginate the request
   optional string after = 3;
+  // Deprecated: This field will be removed in favour of the fields below
+  // More filters may be added to support the legacy ones, if needed
   // Filters to apply to the request
-  map<string, string> filters = 4;
+  map<string, string> filters = 4  [deprecated = true];
```

Comment:
I don't have a problem with depreciating this but when I added it it seemed handy to be able to filter on arbitrary attributes; what is the logic behind removing it?

Reply:
The reason is that it would be nice to have a predefined set of parameters which we can document properly. Right now, the possible keys of this filter are not very obvious and are not documented. However, we won’t be removing the filter until we have all required parameters added and well documented.

Reply:
To be fair, this PR is more about adding filtering by party and tx type (Deposit, Withdraw, Transfer...) than removing stuff. I don't really mind the filters, although it would be nice to have them documented.